test/integration/integration_test_fixtures.py

```diff
@@ -196,6 +247,27 @@ def getTestServerRootUri(self):
     return super(HttpsIntegrationTestBase, self).getTestServerRootUri(True)
 
 
+class MultiServerHttpsIntegrationTestBase(IntegrationTestBase):
+  """
+  Base for https tests against multiple Nighthawk test servers
+  """
+
+  def __init__(self, ip_version, backend_count):
```

Comment:
Same comments about documentation apply to this class.

Reply:
Done